### ISSUE TITLE
Relative vs absolute paths

### ISSUE THREAD

**Issue by vitlok-cyberhawk** [GaryThomsonDev/Cyberhawk#2]
You are checking if file exists using the absolute path, but when you import the file you use relative path and double dots (..)

Is there any reason for this? 

```php

    if (is_file(\dirname(__DIR__).'/config/services.yaml')) {
            $container->import('../config/{services}.yaml');

```

**Comment by GaryThomsonDev** [comment]
I agree, although this is symfony's boiler plate code so i didn't personally write it. It was generated - I'm assuming it expects the directories to remain the same but yea I don't see why they would check it's absolute then use its relative path, seems like a bad way to validate.
